unabridged-works-volume-3/original/12-jews-natural-spiritual.adoc

```diff
@@ -897,15 +931,24 @@ a treasure of life, a well of life,
 out of which the living water will be daily springing up in thee unto life everlasting.
 Amen.
 
+[.signed-section-context-close]
 The 22d of the 7th month, 1676
 
-=== THREE QUERIES UPON THREE VERSES OF THE 46TH PSALM
+=== Three Queries upon Three Verses of the 46th Psalm
 
-Verse 4. "`THERE is a river,
+[quote.scripture, , Verse 4]
```

Comment:
I only put "verse 4" in here and not Psalm 46 too because that's how it is in the QHP document, and it's in the title above. I'm not sure if that's how you would want it though.

Reply:
seems good to me 👍